static/bidder-params/pixfuture.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "PixFuture Adapter Params",
+    "description": "A schema which validates params accepted by the PixFuture adapter",
+    "type": "object",
+    
+    "properties": {
+      "pix_id": {
+        "type": "string",
+        "description": "PixFuture placement ID"
+      }
```

Comment:
Is an empty string `pix_id` considered valid? If not, add a `minLength` field to the `pix_id` object here.

Reply:
The "minLength": 3" have been added.